client/landing/gutenboarding/onboarding-block/animated-placeholder/index.tsx

```diff
@@ -9,7 +9,12 @@ import classnames from 'classnames';
  */
 import './style.scss';
 
-const AnimatedPlaceholder: FunctionComponent = ( { isSlow, texts } ) => {
+interface Props {
+	isSlow?: boolean;
+	texts: Array< string >;
```

Comment:
We're preferring this for simple array types. It means exactly the same thing:

```suggestion
	texts: string[];
```

Reply:
Thanks! TIL!